### PR TITLE
[MRG] Further work

### PR DESCRIPTION
**Pull request by EmyrClement**
# Isolation
Doesn't seem to be applied if you do e.g. electron.electronID("cutBasedElectronID-CSA14-PU20bx25-V0-standalone-tight"), even though this ID specifies isolation requirements.  Modified code to manually apply this.  Not sure if this will change if/when we move to MVA based ID.

# Jet pt thresholds
These have been lowered, so selection criteria can be applied after applying a different set of JEC

# Lepton pt thresholds
Change these to be a few GeV above the trigger thresholds we are currently considering.

### CODE REVIEW

File: python/TopPairElectronPlusJetsSelectionFilter_cfi.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     VertexInputTag = cms.InputTag('offlineSlimmedPrimaryVertices'),
 
     # Lepton cuts
-    minSignalElectronPt=cms.double(30.),
+    minSignalElectronPt=cms.double(34.),
```
Comment (kreczko) [inline python/TopPairElectronPlusJetsSelectionFilter_cfi.py]: BTW: Similar argument to lower jet pT: IF we want to evaluate trigger and/or lepton energy scale, the selection should allow for the necessary wiggle-room

**Comment by EmyrClement** [comment]
Any further comments?

File: plugins/TopPairElectronPlusJetsSelectionFilter.cc [hunk 2]
```diff
@@ -353,12 +358,19 @@ void TopPairElectronPlusJetsSelectionFilter::goodIsolatedElectrons() {
...
 		}
-      	else
-  	        passesIso = true;
+      	else {
+      		double electronSCEta = electron.superCluster()->eta();
+      		if ( electronSCEta <= 1.479 ) {
+				passesIso = relIsoWithDBeta < tightElectronIso_EB_ ? true : false;      			
+      		}
+      		else if ( electronSCEta > 1.479 && electronSCEta < 2.5 ) {
+				passesIso = relIsoWithDBeta < tightElectronIso_EE_ ? true : false;
```
Comment (kreczko) [inline plugins/TopPairElectronPlusJetsSelectionFilter.cc]: Why not
```passesIso = relIsoWithDBeta < tightElectronIso_EB_```
?

File: python/submissionScripts/condor/crab.log [hunk 3]
```diff
@@ -0,0 +1,32 @@
+DEBUG 2015-03-05 21:38:03,369: 	 Executing command: 'status'
```
Comment (kreczko) [inline python/submissionScripts/condor/crab.log]: Do we need crab logs in git?

**Comment by kreczko** [comment]
Looks mainly OK. Could you please remove the crab.log and I will merge it.
The other comments can be addressed later.

**Comment by EmyrClement** [comment]
Done

**Pull request merged**